.env

```diff
@@ -1,6 +1,6 @@
-SIMORGH_BASE_URL=http://localhost:7080
-SIMORGH_ASSETS_MANIFEST_PATH=build/assets-local.json
-SIMORGH_PUBLIC_STATIC_ASSETS_PATH=http://localhost:7080
+SIMORGH_BASE_URL=https://www.test.bbc.com
```

Comment:
Is this right? Shouldnt it be local by default?

Reply:
The assume unchanged didn't happen ... I'll fix